### PR TITLE
Django 1.9 and reversion 1.10 fixes

### PR DESCRIPTION
**Pull request by mikek**
Django 1.9 and reversion 1.10 fixes

 * Tests: create superuser with is_staff flag on
 * Add test case for admin create view

### CODE REVIEW

**Comment by intellectronica** [comment]
:+1:

File: aldryn_reversion/tests/test_admin_views.py [hunk 1]
```diff
@@ -359,6 +360,24 @@ def test_revision_view_reverts_object_to_selected_state(self):
...
+            'admin:{0}_{1}_{2}'.format(
+                obj._meta.app_label,
+                obj._meta.model_name,
+                'add'))
+        client = Client()
+        # TODO: we can replace this with force_login() on Django 1.9+
+        client.login(username=self.super_user.username,
+                     password=self.super_user_password)
+        position = '777'
+        response = client.post(url, data={'position': position}, follow=True)
```
Comment (czpython) [inline aldryn_reversion/tests/test_admin_views.py]: please use existing tool https://github.com/aldryn/aldryn-reversion/blob/master/aldryn_reversion/tests/test_placeholders.py#L43

Comment (mikek) [inline aldryn_reversion/tests/test_admin_views.py]: I do not need a request object, but just to be able to perform a POST request to a view. Also, previously there was a missing is_staff flag on superuser that did not allow him to login into admin, but now he can.

**Comment by czpython** [comment]
:+1:

**Pull request merged**